src/cap-description.adoc

```diff
@@ -258,25 +258,20 @@ references if that SDP bit is set because it "owns" that object.
 |==============================================================================
 
 [#section_cap_sealed]
-==== Sealed (S) Bit
+==== Capability Type (CT) Bit
```

Comment:
I used capability type instead of object type here since I think the former is more consistent with the rest of the specification.

Reply:
I agree in using something other than "o(bject )type", which is a misnomer in the post-CInvoke world.